tools/delete_candidate.php

```diff
@@ -51,12 +51,30 @@
 $candidate = new Candidate();
 $candidate->select($DCCID); //find the candidate with the given DCCID
 
+//Find Issues id with candidate frorign key
+$issueIDs = $DB->pselect("SELECT issueID 
+                FROM issues 
+                WHERE candID=:candID",
+                array('candID' => $DCCID)
+		);
+
+//delete issues_comments
+//delete issues_history
+foreach ($issueIDs as $issueID) {
```

Comment:
This will create a lor of connection to the database. It could be done in the same transaction:
DELETE FROM issues_comments WHERE CandID = :candID